### PR TITLE
[CHAD-1649] Temporary Fix to clear tamper

### PR DESCRIPTION
**Pull request by adityata**
* Tamper is cleared after 10s of it being detected
  This is because tamper is not handled locally

### CODE REVIEW

**Comment by adityata** [comment]
@tpmanley

File: devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy
Comment (tpmanley) [inline devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy]: Should we also remove the clear in `case 0`?

Comment (adityata) [reply devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy]: No, the `case 0` is the proper way for the device to clear reports. Also, there would be no use of any changes as that part of code will not run (as `case 0` would have been handled locally)

**Comment by kianooshkarami** [comment]
Just so I understand the reasoning behind the change, Would you please document the reasoning behind it?

**Comment by adityata** [comment]
* Handling Tamper Attribute locally is not fully possible as we need the extra bit of information as to why the tamper happened (i.e. `descriptionText`). We do not yet have the functionality in Local Execution to set the `descriptionText` for events raised from hub.
  https://github.com/PhysicalGraph/hub-core/pull/972: Tried to address the tamper handling locally, but the `descrptionText` support is not still available. Hence this local change was put on hold.
* By default if `case 3` is not handled locally, it is handled in cloud, thus `tamper=detected` event with `descriptionText` is raised.
  But since `case 0` is handled locally, we will not be able send `tamper=clear` (if tamper alert is not handled locally) and this event will not raised.
* To provide a temoprary fix to this problem, we can send a `tamper=clear` event after some set time (say 10s).

File: devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy
Comment (mckeed) [reply devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy]: We should `unschedule(clearTamper)` before this, in case there are multiple alerts in the time frame.

**Review by mckeed — DISMISSED** [review]

**Review by varzac — DISMISSED** [review]

File: devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy
Comment (adityata) [reply devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy]: `groovy.lang.MissingMethodException: No signature of method: java.util.ArrayList.unschedule() is applicable for argument types: (java.lang.String) values: [clearTamper]`

Getting this exception

**Comment by adityata** [comment]
Can someone please review and merge this? I don't have write access.

**Review by varzac — APPROVED** [review]

**Pull request merged**
